tests/role/env_facts.yml

```diff
@@ -7,7 +7,7 @@
 - name: Set fact for realm name
   set_fact:
     ipaserver_realm: "{{ ipaserver_domain }}  | upper"
-  when: ipaserver_domain is not defined
+  when: ipaserver_realm is not defined
```

Comment:
This is a fix.

Reply:
It shouldn't be here. This should be part of another PR.